presses-universitaires-de-rennes-archeologie-et-culture.csl

```diff
@@ -0,0 +1,471 @@
+      <term name="editor" form="short">
+        <single>éd</single>
+        <multiple>éd</multiple>
+      </term>
+      <term name="collection-editor" form="short">
+        <single>dir.</single>
+        <multiple>dir.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="responsability">
+    <choose>
+      <if variable="version">
+        <group delimiter=" ">
+          <text macro="author"/>
+          <text term="editor" form="short" prefix="(" suffix=")"/>
+        </group>
+      </if>
+      <else-if variable="author editor translator collection-editor" match="any">
+        <choose>
+          <if variable="author">
+            <text macro="author"/>
+          </if>
+          <else-if variable="editor">
+            <text macro="editor2"/>
+          </else-if>
+          <else-if variable="collection-editor">
+            <text macro="collection-editor2"/>
+          </else-if>
+          <else>
+            <text macro="translator"/>
+          </else>
+        </choose>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="author-short">
+    <choose>
+      <if variable="author">
+        <names variable="author">
+          <name form="short" font-weight="bold" et-al-min="3" et-al-use-first="1"/>
+          <label form="short" prefix=", " suffix=". " text-case="uppercase" strip-periods="true"/>
+          <et-al font-weight="bold" font-style="italic"/>
+        </names>
+      </if>
+      <else-if variable="editor">
+        <names variable="editor">
+          <name form="short" font-weight="bold" et-al-min="3" et-al-use-first="1"/>
+          <et-al font-weight="bold" font-style="italic"/>
+        </names>
+      </else-if>
+      <else-if variable="collection-editor">
+        <names variable="collection-editor">
+          <name form="short" font-weight="bold" et-al-min="3" et-al-use-first="1"/>
+          <et-al font-weight="bold" font-style="italic"/>
+        </names>
+      </else-if>
+      <else-if type="book">
+        <text variable="title-short" font-weight="bold"/>
+      </else-if>
+      <else-if type="map">
+        <text variable="title-short" font-weight="bold"/>
+      </else-if>
+      <else-if type="article-journal">
+        <text variable="title-short" font-weight="bold"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="volume">
+    <choose>
+      <if type="book">
+          <group>
+            <text variable="number-of-volumes" prefix="&#160;(" suffix="&#160;"/>
+            <text variable="volume" suffix=")" />
+          </group>
+      </if>
+      <else-if type="map">
+          <group>
+            <text variable="number-of-volumes" prefix="&#160;(" suffix="&#160;"/>
+            <text variable="volume" suffix=")" />
+          </group>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="collection">
+    <text variable="collection-title" prefix="coll. " quotes="true"/>
+    <text variable="collection-number" prefix=", "/>
+  </macro>
```

Comment:
@FrancoisC37, there is another macro with the same "collection" name below, which is not allowed. Can you delete the one you don't need?